src/styles/main.scss

```diff
@@ -1 +1,73 @@
-// put code here
+@import "var.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+.display {
+  position: relative;
+  height: $displaySize;
+  width: $displaySize;
+  margin: auto;
+  border: 5px dotted #000;
+  border-radius: 50%;
+
+  &__minute {
+    position: absolute;
+    bottom: 50%;
+    left: 49.9%;
+    height: $minuteHeight;
+    width: $minuteWidth;
+    background-color: #f09;
+    animation-name: move;
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
+
+    @keyframes move {
+      from {
+        transform: rotate(0deg);
+        transform-origin: bottom;
+      }
+
+      to {
+        transform: rotate(360deg);
+        transform-origin: bottom;
+      }
+    }
+  }
+
+  &__hour {
+    position: absolute;
+    bottom: 50%;
+    left: 48.5%;
```

Comment:
try to work with positioning and transforming
<img width="689" alt="Screenshot 2023-12-16 at 15 29 56" src="https://github.com/mate-academy/layout_stop-watch/assets/117587463/e48dcf84-50b0-4b95-852a-4fd81c023b56">